macro/CheckClusters_mft.C

```diff
@@ -53,7 +53,8 @@ void CheckClusters_mft(Int_t nEvents = 10, Int_t nMuons = 200)
   Char_t filename[100];
 
   // Geometry
-  o2::base::GeometryManager::loadGeometry("O2geometry.root", "FAIRGeom");
+  sprintf(filename, NameConf::getGeomFileName("o2sim").c_str());
```

Comment:
I think we can add ``FAIRGeom`` to NameConf, and change the ``o2::base::GeometryManager::loadGeometry`` to
````
static void loadGeometry(std::string_view geomFileName = "")
{
   std::string name = geomFileName.empty ? NameConf::getGeomFileName() : geomFileName();
   TFile flGeom( name.c_str() );
   ...
}
````

Reply:
yes absolutely